app/assets/stylesheets/blog.css.scss

```diff
@@ -0,0 +1,10 @@
+//
```

Comment:
I see that this PR is dependent on the blog related one. This is something that is great to note in the PR description, and helps prevent the too-early merging of dependent branches.